src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,89 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.UUID;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class BasicValidationServiceTest {
+
+    BasicValidationService basicValidation;
+
+    @BeforeEach
+    void setUp() {
+        basicValidation = new BasicValidationService();
+    }
+
+    @Test
+    void validateAmountIsNull_shouldThrowAnException() {
+        assertThrows(IllegalArgumentException.class, () -> basicValidation.validateAmount(null))
+                .getMessage()
+                .equalsIgnoreCase("Amount must not be null");
+    }
+
+    @Test
+    void validateAmountIsSmallerThanZero_shouldThrowAnException() {
+        assertThrows(IllegalArgumentException.class, () -> basicValidation.validateAmount(-1.0))
+                .getMessage()
+                .equalsIgnoreCase("Amount must be greater than 0");
+    }
+
+    @Test
+    void validateAmountIsGreaterThanZero_shouldNotThrowAnException() {
+        assertDoesNotThrow(() -> basicValidation.validateAmount(2.0));
+    }
+
+    @Test
+    void validatePaymentIdIsNull_shouldThrowAnException() {
+        assertThrows(IllegalArgumentException.class, () -> basicValidation.validatePaymentId(null))
+                .getMessage()
+                .equalsIgnoreCase("Payment id must not be null");
+    }
+
+    @Test
+    void validatePaymentIdIsNotNull_shouldNotThrowAnException() {
+        UUID randomID = UUID.randomUUID();
+        assertDoesNotThrow(() -> basicValidation.validatePaymentId(randomID));
+    }
+
+    @Test
+    void validateUserIdIsNull_shouldThrowAnException() {
+        assertThrows(IllegalArgumentException.class, () -> basicValidation.validateUserId(null))
+                .getMessage()
+                .equalsIgnoreCase("User id must not be null");
+    }
+
+    @Test
+    void validateUserIdIsNotNull_shouldNotThrowAnException() {
+        assertDoesNotThrow(() -> basicValidation.validateUserId(1));
+    }
+
+    @Test
+    void validateUserStatusIsNotActive(){
+        User user = new User(1, "Ion", Status.INACTIVE);
+        assertThrows(IllegalArgumentException.class, () -> basicValidation.validateUser(user))
+                .getMessage()
+                .equalsIgnoreCase("User with id " + user.getId() + " not in ACTIVE status");
```

Comment:
This is a example of how you can reformat:
 ```
 final IllegalArgumentException illegalArgumentException = assertThrows(IllegalArgumentException.class, () -> basicValidation.validateUser(user));
        assertEquals(illegalArgumentException.getMessage(),"User with id " + user.getId() + " not in ACTIVE status");
```

Reply:
Done